docs/tutorial.md

```diff
@@ -152,7 +148,7 @@ spec:
 
 The following is a `Task` with inputs and outputs. The input resource is a
 GitHub repository and the output is the image produced from that source. The
-args of the task command support variable substitution so that the definition of task is
+args of the `Task` command supports variable substitution so that the definition of `Task` is
```

Comment:
`The args ... support` is correct I think?

Reply:
Oh true